libs/server/core/application-services/src/lib/activity-post/activity-post.service.ts

```diff
@@ -1,22 +1,33 @@
 import { Injectable } from "@nestjs/common";
-import { ActivityPostRepository, LikeRepository } from "@involvemint/server/core/domain-services";
-import { ActivityPost, ActivityPostQuery, CreateActivityPostDto, DisableActivityPostDto, EnableActivityPostDto, LikeActivityPostDto, likeQuery, UnlikeActivityPostDto } from "@involvemint/shared/domain";
-import { IQuery } from "@orcha/common";
+import { ActivityPostRepository, LikeRepository, UserRepository } from "@involvemint/server/core/domain-services";
+import { IPaginate, IParserObject, IQuery } from "@orcha/common";
 import { AuthService } from '../auth/auth.service';
 import * as uuid from 'uuid';
+import { IQueryObject } from "@orcha/common/src/lib/query";
+import { ActivityPost, ActivityPostQuery, CreateActivityPostDto, DigestActivityPostDto, DisableActivityPostDto, EnableActivityPostDto, GetActivityPostDto, LikeActivityPostDto, likeQuery, RecentActivityPostDto, UnlikeActivityPostDto } from "@involvemint/shared/domain";
+import { Cron, CronExpression } from "@nestjs/schedule"
+import { SMSService } from "../sms/sms.service";
+import { UserQuery } from '@involvemint/shared/domain';
 
 @Injectable()
 export class ActivityPostService {
     constructor(
         private readonly auth: AuthService,
         private readonly activityPostRepo: ActivityPostRepository,
-        private readonly likeRepo: LikeRepository
+        private readonly likeRepo: LikeRepository,
+        private readonly sms: SMSService,
+        private readonly user: UserRepository
     ) {}
 
-    async list(query: IQuery<ActivityPost[]>, token: string) {
+    async list(query: IQuery<ActivityPost[]>, token: string, dto: RecentActivityPostDto) {
```

Comment:
@AlexanderChiuy is it possible for us to revert this addition to 'list' API since we don't use it at all anymore?